app/controllers/job_templates_controller.rb

```diff
@@ -41,11 +41,6 @@ def import
     end
   end
 
-  def export
```

Comment:
should be removed from `routes.rb`: this is the reason why the tests are failling on permission check